### ISSUE TITLE
Can't disable flip on click

### ISSUE THREAD

**Issue by geraldlnj** [kgrandemange/vue-flip#20]
active-click seems to always be enabled, even when I explicitly set it to false. 

I tried your codepen in #17 , https://codesandbox.io/s/vue-template-lmvtj, and when I delete the active-click, it words as expected,  I can only toggle the flip using the button. 

When I try it in my project (https://geraldlnj.github.io/cards-against-clones/), I can still click on them. Any ideas? I even tried copy/pasting the code from your codepen to my project and the same thing happened.

Here's my code as reference.

```Vue
<template>
  <vue-flip
    v-model="flipped"
    height="384px"
    width="240px"
  >
    <template v-slot:front>
      <CardFace v-bind:type="type"/>
    </template>
    <template v-slot:back>
      <CardBack :type="type" :text="text"/>
    </template>
  </vue-flip>
</template>

<script lang="ts">
import Vue from 'vue';
// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
// @ts-ignore
import VueFlip from 'vue-flip';
import CardBack from '@/components/CardBack.vue';
import CardFace from '@/components/CardFace.vue';


export default Vue.extend({
  name: 'Card',
  components: {
    VueFlip,
    CardBack,
    CardFace,
  },
  props: {
    type: {
      default: 'question',
      type: String,
      required: true,
    },
    text: {
      default: 'Placeholder Question',
      type: String,
      required: true,
    },
  },

  data: () => ({
    flipped: false,
  }),
  computed: {
    classObject() {
      return {
        card: true,
        question: this.type === 'question',
      };
    },
  },
  methods: {
  },
});
</script>

```

**Comment by kgrandemange** [comment]
My bad 🙁. I fixed it but I don't publish a new version...
Can you confirm your problem is solved with the 0.4.4 version ?

Here an example:

https://codesandbox.io/s/broken-worker-8fgls?file=/src/App.vue

**Comment by geraldlnj** [comment]
Yup, looks like it's fixed! Thanks!
